warp/src/js_exports/raygun.rs

```diff
@@ -753,24 +753,37 @@ impl Message {
 #[wasm_bindgen]
 pub struct AttachmentFile {
     file: String,
-    stream: Option<web_sys::ReadableStream>,
+    stream: Option<AttachmentStream>,
 }
-
 #[wasm_bindgen]
 impl AttachmentFile {
     #[wasm_bindgen(constructor)]
-    pub fn new(file: String, stream: Option<web_sys::ReadableStream>) -> Self {
+    pub fn new(file: String, stream: Option<AttachmentStream>) -> Self {
         Self { file, stream }
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[wasm_bindgen]
+pub struct AttachmentStream {
+    size: usize,
```

Comment:
We should probably make this type into `Option<usize>` and pass that option through to `Location::Stream`